client/jetpack-onboarding/main.jsx

```diff
@@ -0,0 +1,56 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { compact } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Wizard from 'components/wizard';
+import {
+	JETPACK_ONBOARDING_COMPONENTS as COMPONENTS,
+	JETPACK_ONBOARDING_STEPS as STEPS,
+} from './constants';
+
+class JetpackOnboardingMain extends React.Component {
```

Comment:
Let's make this a `PureComponent`. (I know `connect()` will turn it into one anyway, but I like to explicitly state it whenever possible).

Reply:
Done 👍 8547b85e6cade12ff5ea0a70b10c94b40bcfce8a